src/utils/dato.ts

```diff
@@ -97,3 +97,10 @@ export const erPeriodeGyldig = (periode: IPeriode | undefined): boolean => {
 export const erEnMånedTilbakeITid = (dato: string): boolean => {
   return !isAfter(strengTilDato(dato), addMonths(dagensDato, -1));
 };
+
+export const nåværendeÅr = new Date().getFullYear();
+
+export const erMellomStartenAvMaiOgSluttenAvAugust = () => {
+  const nåværendeDato = new Date();
+  return nåværendeDato.getMonth() >= 4 && nåværendeDato.getMonth() <= 7;
```

Comment:
```suggestion
export const erDagensDatoMellomMåneder = (månedFra: number, månedTil: number) => {
  const nåværendeDato = new Date();
  return nåværendeDato.getMonth() >= månedFra && nåværendeDato.getMonth() <= månedTil;
```

Ville vurdert å gjøre koden mer gjenbrukbar ved å sende inn månedene man skal være mellom.

Reply:
Har gjort koden mer gjenbrukbar nå, det er heller ikke nødvendig å ta høyde for index lenger.